playgrounds/jest-cjs/jest.config.bundler.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @type {import("jest").Config}
+ **/
+module.exports = {
+  testEnvironment: 'node',
+  transformIgnorePatterns: [
+    // 'node_modules/(?!@faker-js).+', // required if the projects package manager is "npm"
+    'node_modules/.pnpm/.+/node_modules/(?!@faker-js).+', // required if the package manager is "pnpm"
+    // I have no idea how yarn builds the node_modules directory otherwise the pattern would be here ¯\(o_o)/¯
+    //
+    // NOTE!!!
+    // Exactly one of  must be active at the same time - no more, no less!!!
```

Comment:
one of what?

Reply:
totally fine 😆 
I wanted to give you a bit of crazy-freedom, but yeah, the used language is a bit _interesting_ XD